languages.toml

```diff
@@ -2635,3 +2636,17 @@ file-types = ["webc"]
 roots = []
 indent = { tab-width = 2, unit = "  " }
 grammar = "html"
+
+[[language]]
+name = "typst"
+scope = "source.typst"
+injection-regex = "typst"
+file-types = ["typst", "typ"]
+roots = []
+comment-token = "//"
+language-servers = ["typst-lsp"]
+indent = { tab-width = 4, unit = "    " }
```

Comment:
FYI, the typst docs and online editor both use 2 spaces for indentation, not 4. [Here's an example.](https://typst.app/docs/tutorial/advanced-styling/#set-rules)

Reply:
Thanks! That's true, I've fixed it 👍🏻